lib/holidays.rb

```diff
@@ -9,44 +9,6 @@
 require 'holidays/errors'
 require 'holidays/load_all_definitions'
 
-# == Region options
```

Comment:
I just thought maybe it was a mistake since it was committed with this patch. I guess it's covered in the readme. I think it's fine for this project with not so many methods, for larger projects like Rails I think inline commenting is useful.